src/components/CurrencySwitcher/CurrencySwitcher.tsx

```diff
@@ -36,7 +38,10 @@ const AccountActionButton = ({ balance, isDemo }: AccountActionButtonProps) => {
                         loginid: activeLoginid,
                     });
                 } else {
-                    navigate('/cashier/deposit');
+                    if (currencyConfig?.currency?.isCrypto)
+                        startPerformanceEventTimer('load_crypto_deposit_cashier_time');
+                    else startPerformanceEventTimer('load_fiat_deposit_cashier_time');
+                    navigate('/cashier/deposit#deposit');
```

Comment:
As traders hub is in new repo, our url is changed too. We need to use this url
https://app.deriv.com/cashier/deposit#deposit